release-notes/SRM/SRM_1.2.35.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: SRM_1.2.35
+---
+
+# SRM 1.2.35
+
+> [!NOTE]
+> This version requires that **DataMiner 10.3.2.0 – 12627 or higher** is installed. It is not compatible with the DataMiner Main Release track.
+
+## Enhancements
+
+#### Skyline Booking Monitoring: Failed states added [ID_38271]
+
+The following Failed states have been added to the Skyline Booking Monitoring connector: Failed Service Pre-Roll, Failed Service Active, Failed Service Post-Roll, and Failed Complete.
```

Comment:
....Skyline Booking Monitoring connector so it's up to date with the Skyline Booking Manager connector...

Reply:
Done!